sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -472,7 +473,7 @@ export type SqlRuleAction = {
 
 // @public
 export interface SqlRuleFilter {
-    sqlExpression?: string;
+    sqlExpression: string;
```

Comment:
@HarshaNalluru, shouldnt we be making the same change to `SqlRuleAction`? Or do we want `SqlRuleAction` to only have optional properties so that we can represent the scenario of no action?

Reply:
Creating an SQLRuleFilter without an sqlExpression since it would mean that the filter is being created without a rule which doesn't make a lot of sense, hence intending to make `sqlExpression` mandatory for SQLRuleFilter.

The same is not applicable for the rule action, which takes effect after the message is filtered with the rule filters making no rule action a valid scenario. The service supports the EmptyRuleAction which allows `sqlExpression` to be optional for SQLRuleAction.

Reply:
makes sense, thanks!